api-report/datastore.api.md

```diff
@@ -103,6 +103,8 @@ export class FluidDataStoreRuntime extends TypedEventEmitter<IFluidDataStoreRunt
     resolveHandle(request: IRequest): Promise<IResponse>;
     reSubmit(type: DataStoreMessageType, content: any, localOpMetadata: unknown): void;
     // (undocumented)
+    rollback?(type: DataStoreMessageType, content: any, localOpMetadata: unknown): void;
```

Comment:
@anthony-murphy, should I remove type from the rollback parameters? I don't think we'll ever support rolling back attach, but maybe there will be more types in the future?

Reply:
l like the type, as it allows safer narrowing of the content, and will let us ensure we strictly handles that types we want to support, and no new types can sneak in to cause problems